#! /bin/sh /usr/share/dpatch/dpatch-run
## no_pdf_in_print.dpatch by Thomas Weber <thomas.weber.mail@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Warning about missing pdflib
## DP: #478677

@DPATCH@
diff -urNad octave3.0-3.0.1~/scripts/plot/print.m octave3.0-3.0.1/scripts/plot/print.m
--- octave3.0-3.0.1~/scripts/plot/print.m	2008-04-21 18:00:20.000000000 +0200
+++ octave3.0-3.0.1/scripts/plot/print.m	2008-05-21 12:17:20.000000000 +0200
@@ -93,6 +93,9 @@
 ##     Scalable vector graphics
 ##   @item pdf
 ##     Portable document format
+##
+##     NOTE: The gnuplot binary as shipped by Debian cannot create PDF files, see http://bugs.debian.org/478677
+##
 ##   @end table
 ##
 ##   Other devices are supported by "convert" from ImageMagick.  Type
